backend/tilavarauspalvelu/management/commands/data_creation/create_reservations.py

```diff
@@ -191,25 +194,32 @@ def _create_normal_reservations(
         applying_for_free_of_charge: bool = False
         free_of_charge_reason: str | None = None
         deny_reason: ReservationDenyReason | None = None
-
-        if reservation_unit.require_reservation_handling:
-            reservation_state = weighted_choice(handling_state_choices, weights=[2, 5, 1])
-            if reservation_state in {ReservationStateChoice.CONFIRMED, ReservationStateChoice.DENIED}:
-                handled_at = local_datetime()
+        handled_payment_due_by: datetime.datetime | None = None
 
         for reservation_date in get_date_range(start_date, number=14):
+            if reservation_unit.require_reservation_handling:
+                reservation_state = next(handling_state_choices_cycle)
```

Comment:
Why not use `weighted_choice(handling_state_choices, [2, 5, 1])` here? 🤔

Reply:
Just a bit less randomness. Should be mostly the same.